azure/public.go

```diff
@@ -0,0 +1,347 @@
+package azure
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Azure/azure-storage-blob-go/azblob"
+	"github.com/aptly-dev/aptly/aptly"
+	"github.com/aptly-dev/aptly/utils"
+	"github.com/pkg/errors"
+)
+
+// PublishedStorage abstract file system with published files (actually hosted on Azure)
+type PublishedStorage struct {
+	container azblob.ContainerURL
+	prefix    string
+	pathCache map[string]string
+}
+
+// Check interface
+var (
+	_ aptly.PublishedStorage = (*PublishedStorage)(nil)
+)
+
+// NewPublishedStorage creates published storage from Azure storage credentials
+func NewPublishedStorage(accountName, accountKey, container, prefix string) (*PublishedStorage, error) {
+	credential, err := azblob.NewSharedKeyCredential(accountName, accountKey)
+	if err != nil {
+		return nil, err
+	}
+
+	containerURL, err := url.Parse(fmt.Sprintf("https://%s.blob.core.windows.net/%s", accountName, container))
+	if err != nil {
+		return nil, err
+	}
+
+	result := &PublishedStorage{
+		container: azblob.NewContainerURL(*containerURL, azblob.NewPipeline(credential, azblob.PipelineOptions{})),
+		prefix:    prefix,
+	}
+
+	return result, nil
+}
+
+// String
+func (storage *PublishedStorage) String() string {
+	return fmt.Sprintf("Azure :%s/%s", storage.container, storage.prefix)
```

Comment:
```suggestion
	return fmt.Sprintf("Azure: %s/%s", storage.container, storage.prefix)
```
(this is the minor change I had on my branch)

Reply:
No. It is not a typo. I was trying to follow the format convention set by other storage back-ends in aptly.

https://github.com/aptly-dev/aptly/blob/master/s3/public.go#L124
https://github.com/aptly-dev/aptly/blob/master/swift/public.go#L112